### DISCUSSION TITLE
is it possible to exclude trait at all?

### DISCUSSION THREAD

**Discussion by vergara111** [nftchef/art-engine#121] [Help + Q&A]
Hi, @nftchief!

is it possible to completely exclude the appearance of one of the folders from layerConfigurations in traits? I mean, so that the folder that is in layerConfigurations does not appear in traits at all.

**Comment by nftchef** [comment]
Hi!
Can you explain the setup a little bit more? if you do not want a folder to appear, then usually it would not be included in the list, so I must be missing something.

**Reply by vergara111** [reply]
I mean I want images from a specific folder to be used during generation, but I don't want there to be a trait with the this folder name at all.

**Reply by nftchef** [reply]
if I understand correctly, the remove trait util is the easiest solution, used _after_ the generator it done.
https://github.com/nftchef/art-engine#remove-trait-util

**Reply by vergara111** [reply]
exactly what I need, forgot about this function, thank you!
